code/modules/mob/living/basic/space_fauna/spider/giant_spider/giant_spider_variants.dm

```diff
@@ -18,6 +47,41 @@
 	player_speed_modifier = -3.1
 	menu_description = "Fast spider variant specializing in catching running prey and toxin injection, but has less health and damage."
 
+/mob/living/basic/giant_spider/hunter/Initialize(mapload)
+	. = ..()
+
+	AddElement(/datum/element/web_walker, /datum/movespeed_modifier/hunter_web)
```

Comment:
Rather than add this individually to almost every single spider variant, make a var on `/mob/living/basic/giant_spider` which contains the movement speed modifier to apply and then run this code in the `Initialize` of the base class if it is not null.